src/runtime/runtime.cs

```diff
@@ -2129,7 +2145,7 @@ internal static void Py_CLEAR(ref IntPtr ob)
         //====================================================================
 
         [DllImport(_PythonDll, CallingConvention = CallingConvention.Cdecl)]
-        internal static extern IntPtr PyCapsule_New(IntPtr pointer, string name, IntPtr destructor);
+        internal static extern NewReference PyCapsule_New(IntPtr pointer, string name, IntPtr destructor);
 
         [DllImport(_PythonDll, CallingConvention = CallingConvention.Cdecl)]
         internal static extern IntPtr PyCapsule_GetPointer(IntPtr capsule, string name);
```

Comment:
This should be `IntPtr PyCapsule_GetPointer(BorrowedReference capsule, string name)`

Reply:
Or you can add an overload.